### PR TITLE
several talks and papers

### PR DESCRIPTION
**Pull request by cranmer**

### CODE REVIEW

**Comment by cranmer** [comment]
Hmmm... not clear why it failed:
```

Check for added large files..............................................Passed
Mixed line ending........................................................Passed
Trim Trailing Whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing _data/events/20210906-MODE-Differentiable-Programming.yml
Fixing _data/people/cranmer.yml
```

File: _data/people/cranmer.yml [hunk 1]
```diff
@@ -540,10 +540,110 @@ presentations:
...
+  meetingurl: https://indico.cern.ch/event/1051224/
+  focus-area: as
+  location: Virtual
+- title: "Panel: Sins and marvels of AI research"
+  date: 2021-11-06
+  url: https://conference2021.mlinpl.org
+  meeting: ML in PL (Machine Learning in Poland)
+  meetingurl: https://conference2021.mlinpl.org
+  focus-area: as
+  location: Virtual
```
Comment (alexander-held) [inline _data/people/cranmer.yml]: This has a trailing whitespace.

File: _data/people/cranmer.yml [hunk 2]
```diff
@@ -540,10 +540,110 @@ presentations:
...
+  meetingurl: http://www.ipam.ucla.edu/programs/workshops/workshop-iii-source-inference-and-parameter-estimation-in-gravitational-wave-astronomy/
+  focus-area: as
+  location: Institute for Pure and Applied Mathematics (IPAM)
+- title: "Inductive bias: lessons learned from physics"
+  date: 2021-12-13
+  url: https://physical-reasoning.github.io/speakers/
+  meeting: Physical Reasoning and Inductive Biases for the Real World workshop at NeurIPS 2021
+  meetingurl: https://physical-reasoning.github.io
+  focus-area: as
+  location: Virtual
```
Comment (alexander-held) [inline _data/people/cranmer.yml]: This line also has a trailing whitespace.

**Comment by cranmer** [comment]
I see that there were trailing white spaces, but given the name
`Trim Trailing Whitespace.................................................Failed`
It looks like that is going to trim and modify the file to remove the trailing whitespace. 
It seems like it failed to fix it for some reason, but it's not clear why.

**Comment by alexander-held** [comment]
I think it would edit the files when run locally, but it won't push the changes (http://pre-commit.ci/ does I believe), so in this case it has to be fixed manually.

edit: I seem to be wrong, see Henry's comment below, this is actually using pre-commit.ci

**Comment by henryiii** [comment]
It should have automatically committed a fix. I'm asking to see if this is a known issue.

**Comment by cranmer** [comment]
Well that's weird... I fixed whitespace in my files. Now it's failing due to 
`Fixing _data/events/20210906-MODE-Differentiable-Programming.yml`
But that file is not in my PR. Was it cached?

**Comment by alexander-held** [comment]
That file still has a trailing whitespace leftover on master from https://github.com/iris-hep/iris-hep.github.io-source/commit/2ed60c8b8de8a40b8e23899e568d2cd1c55ab523, in the `description` field.

**Comment by henryiii** [comment]
CC @pelmer. I bet that's why pre-commit is not fixing it automatically - it doesn't want to push a fix to a file that was not included in this PR. That's my guess, anyway.

**Pull request merged**